app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<p> User name with posts </p>
\ No newline at end of file
+<h2> User name with posts </h2>
+
+<div class = 'user-content'> 
+<div class = 'user'>
+<div class = "image-placeholder"></div>
+<div class = 'user-block'>
+ <h3><%= @user.name %></h3>
+   <p > Number of posts: <%= @user.posts.count%> </p>
+ </div>
+</div>
+ <div class ='user-bio'>
+ <p><strong>Bio </strong> </p>
+ <p> <%= @user.bio %> </p>
+  </div>
+  <div class = 'posts-content'>
+     <% @posts.each do |post|%>
+     <div class = 'post-content'> 
+     <li><h4>   <%= link_to post.title, user_posts_path(@user) %>
+    </h4></li>
+     <p> <%= post.text%> </p>
+     <p class = 'counters'> Comments: <%= post.comments_counter.to_i > 0 ? post.comments_counter : '0'  %> Likes : <%= post.likes_counters.to_i > 0 ? post.likes_counters : '0'%>  </p>
+     </div>
+     <%end%>
+   </div>
+<div>
```

Comment:
- [ ] Hi @AmineDerbal, it seems you are missing the **See all Post** button at the button of this page. I suggest you include the button. Please don't forget that clicking on the button links opens the user's post index page.